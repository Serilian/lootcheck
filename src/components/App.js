import React from 'react';
import Wallet from "./Wallet";
import Loot from './Loot';

function App() {
    return (
        <div>
            <h2>Loot Check</h2>
            <Wallet/>
            <hr/>
            <Loot/>
            <div className={'coin-link'}>Powered by <a href="https://www.coindesk.com/price" target={'_blank'}>CoinDesk</a></div>
        </div>
    );
}


export default App;