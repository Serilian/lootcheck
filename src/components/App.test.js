import React from 'react';
import App from './App.js';
import {shallow} from 'enzyme';


describe("Given App component", () => {
    let app;

    beforeEach(() => {
        app = shallow(<App/>);
    });

    it('renders properly', () => {
        expect(app.debug()).toMatchSnapshot();
    });

    it("contains a connected Wallet component", () => {
        expect(app.find("Connect(Wallet)").exists()).toBe(true);
    });

``});