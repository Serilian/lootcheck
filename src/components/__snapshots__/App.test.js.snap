// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given App component renders properly 1`] = `
"<div>
  <h2>
    Loot Check
  </h2>
  <Connect(Wallet) />
</div>"
`;
